features/import.feature

```diff
@@ -138,3 +138,64 @@ Feature: Import products
     """
     [ '[row 0] Product deleted.' ]
     """
+
+    Scenario: Batch import
+
+      Given a file named "i.csv" with:
+      """
+      id,productType,slug.en,variantId,name,sku,attr-text-n
+      0912,ImpEx with all types,slug_1,1,myProduct,1,key_1
```

Comment:
The same id for all products does not make sense here.
Simply remove the column.